.github/workflows/tests.sh

```diff
@@ -9,9 +9,13 @@ export PYTHONPATH="${PYTHONPATH}:${GITHUB_WORKSPACE}/.ci"
 # - pytest-cov configuration taken from top-level .coveragerc
 # - coverage is reported as XML and in terminal,
 #   including the numbers/ranges of lines which are not covered
-# - coverage results of multiple tests are collected
+# - coverage results of multiple tests are collected (this is always the case with Codecov)
```

Comment:
I think the comment here refers to coverage results within a _single_ workflow step (when `pytest` is run more than once). That's what ``--cov-append`` does, otherwise the coverage file would be overwritten (AFAIK, _before_ it's uploaded to codecov).

Reply:
For sure.
I also read, however, that one could simply upload it (using `coverage` CLI) to Codecov, as soon as you have a "new" report, and Codecov would _always_ merge these reports on their side.
However, I much prefer adding this here and not continuously upload to Codecov ...
But maybe I should rewrite my addition to the comment here to represent the intention better.